src/main/java/core/basesyntax/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line after the class declaration.